android/libraries/rib-base/src/main/java/com/badoo/ribs/core/routing/action/CompositeRoutingAction.kt

```diff
@@ -1,20 +1,20 @@
 package com.badoo.ribs.core.routing.action
 
-import android.os.Bundle
 import com.badoo.ribs.core.Node
-import com.badoo.ribs.core.routing.portal.AncestryInfo
+import com.badoo.ribs.core.builder.BuildContext
 
 class CompositeRoutingAction(
     private vararg val routingActions: RoutingAction
 ) : RoutingAction {
 
+    override val nbNodesToBuild: Int = routingActions.asList().fold(0) { acc, r -> acc + r.nbNodesToBuild }
```

Comment:
reduce?
also array has fold/reduce extensions as well :)

Reply:
Yeah, IDE had issues with array extensions, not sure why.